views/profile.ejs

```diff
@@ -0,0 +1,172 @@
+<body>
+    <header class="jumbotron">
+        <div class="container marketing">
+            <div class="row row-header">
+                <div class="col-12 col-sm-6">
+                    <h1><%- first -%> <%- last -%></h1>
+                    <p><%- email -%></p>
+                </div>
+                <div class="col-12 col-sm">
+                </div>
+            </div>
+        </div>
+
+    </header>
+
+    </div>
+    <div class="container">
+
+        <div class="row">
+
+            <div class="col-sm-4">
+
+                <div class="card">
+                    <form method="POST" action="https://preprlabs.org/pictureChange" accept-charset="UTF-8" id="form_image" enctype="multipart/form-data">
```

Comment:
Is the form element required in this case? And the action attribute is pointing to an external address. Hence, it should be removed.